addons/share-through-mystuff/addon.json

```diff
@@ -20,6 +20,14 @@
       "runAtComplete": true
     }
   ],
+  "settings": [
+    {
+      "name": "Ask if the project should be shared",
```

Comment:
`confirm-actions` already has a confirmation setting for sharing projects.
https://github.com/ScratchAddons/ScratchAddons/blob/48106c8599fca407ded70aeb073085cd7b6b655a/addons/confirm-actions/addon.json#L22
This addon _could_ use that setting instead (so we have just one setting, easy to use), but then users may want to not/only have a confirmation for sharing **from the My Stuff page**. Worth discussing.

Reply:
IMO, this new addon shouldn't have any settings. It should always ask to confirm.
If you want to share many projects rapidly without confirmation, just open 1 tab per project.

Reply:
One tab per project gets easily cluttered--if it's more than ~2 and makes the process of sharing a project 10% slower when you have a slower connection.

Reply:
I don't think we should take into account the potential need to share multiple projects at once. This is Scratch Addons, not Scratch Macros.

Reply:
fine.